generator/generator.go

```diff
@@ -247,9 +248,18 @@ func (img *Image) appendInitBinary(initBinary string) error {
 
 func (img *Image) appendExtraFiles(binaries []string) error {
 	for _, f := range binaries {
-		if !strings.HasPrefix(f, "/") {
-			// simple names like "strace" are resolved as binaries under /usr/bin
-			f = "/usr/bin/" + f
+		if !filepath.IsAbs(f) {
+			// If the given name is not an absolute path, assume that it refers
+			// to an executable and lookup the path to the executable using $PATH.
+			var err error
+			f, err = exec.LookPath(f)
+			if err != nil {
+				return err
+			}
+
+			if err := img.AppendFileTo(f, "/usr/bin"); err != nil {
```

Comment:
you have 2 `AppendFile()` for the same file `f` inside the loop. I think it should be called only once.

Reply:
Good catch, fixed.